documentation/_ext/link_usernames.py

```diff
@@ -4,47 +4,115 @@
 and: https://stackoverflow.com/a/31924901/3277713  CC BY-SA 3.0 by C_Z_
 
 This extension replaces username references of the form `@username` with a link
-to the user's GitHub profile. In order to prevent adding references to existing
-existing links (e.g. `[@username](...)`, whitespace is required in front of the `@`.
+to the user's GitHub profile.
 
-The GITHUB_IGNORE_USERNAMES set contains usernames that should not be linked but may
-appear to be usernames in the documentation.
+The plugin ignores code inside of fixed text blocks, including code blocks and backticks.
 """
 import re
 
+from docutils import nodes
 from sphinx.application import Sphinx
+from sphinx.transforms import SphinxTransform
 from sphinx.util import logging
 
 
 logger = logging.getLogger(__name__)
 
 # Format to use for the link to the GitHub profile
-GITHUB_USERNAME_TEMPLATE = "https://github.com/{}"
+GITHUB_USERNAME_TEMPLATE = "https://github.com/{username}"
+# Format to use for team mentions
+GITHUB_TEAM_TEMPLATE = "https://github.com/orgs/{org}/teams/{team}"
+
 # Regex to match username references
 GITHUB_USERNAME_REGEX = re.compile(
     r"""
-\ # Required initial whitespace
-@([A-Za-z0-9-]+)  # Match @ then any alphanumeric character or - for the username
+@(?P<username>[A-Za-z0-9-]+)  # Match @ then any alphanumeric character or - for the username
+(?P<team>/[A-Za-z0-9-]+)?  # If this is a team reference, match the team separately
```

Comment:
At the risk of slightly complicating the regex, we can keep the leading `/` out of the team group. Also `A-Za-z0-9` can be simplified as `\w`.

```suggestion
@(?P<username>[\w-]+)  # Match @ then any alphanumeric character or - for the username
(/(?P<team>[\w-]+))?  # If this is a team reference, match the team separately
```

Also TIL comments work inside triple-quoted strings!

Reply:
Yeah! But only with the verbose flag, mind: https://docs.python.org/3/library/re.html?highlight=re%20verbose#re.VERBOSE